### PR TITLE
Remove template code for runc and containerd

### PR DESCRIPTION
**Pull request by hqhq**
**\- What I did**

**\- How I did it**

**\- How to verify it**

**\- A picture of a cute animal (not mandatory but encouraged)**

Signed-off-by: Qiang Huang h.huangqiang@huawei.com

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -255,7 +255,8 @@ RUN set -x \
 	&& cd "$GOPATH/src/github.com/opencontainers/runc" \
 	&& git checkout -q "$RUNC_COMMIT" \
 	&& make static BUILDTAGS="seccomp apparmor selinux" \
-	&& cp runc /usr/local/bin/docker-runc
+	&& cp runc /usr/local/bin/docker-runc \
+	&& rm -rf "$GOPATH"
```
Comment (coolljt0725) [inline Dockerfile]: I think this is not necessary since the the directory is made by `mktmp`, it's a temporary file

Comment (hqhq) [inline Dockerfile]: Just keep it consistent with all other components.

Comment (coolljt0725) [inline Dockerfile]: make sense, `/tmp` is not mount as `tmpfs`  during the build, we have to move the tmp file manually

**Comment by tiborvass** [comment]
LGTM

**Comment by vdemeester** [comment]
LGTM 🐮

**Pull request merged**